deu/paris-match-cleared.vbs

```diff
@@ -299,73 +319,63 @@ IF send_notice_checkbox = CHECKED THEN
-    	IF proof_residency_CHECKBOX = CHECKED THEN pending_verifs = pending_verifs & "Residency, "
-    	IF schl_verf_CHECKBOX = CHECKED THEN pending_verifs = pending_verifs & "School, "
-    	'trims excess spaces of pending_verifs
-    	pending_verifs = trim(pending_verifs)
-    	'takes the last comma off of pending_verifs when autofilled into dialog if more more than one app date is found and additional app is selected
-    	IF right(pending_verifs, 1) = "," THEN pending_verifs = left(pending_verifs, len(pending_verifs) - 1)
-    	Due_date = dateadd("d", 10, date)	'defaults the due date for all verifications at 10 days
-
-    	'-----------------------------------------------------------------------The case note
-    	start_a_blank_CASE_NOTE
-    	CALL write_variable_in_CASE_NOTE ("-----" & Match_month & " PARIS MATCH " & "(" & first_name &  ") DIFF NOTICE SENT-----")
-    	CALL write_bullet_and_variable_in_CASE_NOTE("Client Name", Client_Name)
-    	CALL write_bullet_and_variable_in_CASE_NOTE("MN Active Programs", MN_active_programs)
-    	'formatting for multiple states
-    	For item = 0 to Ubound(state_array, 2)
-    		CALL write_variable_in_CASE_NOTE("----- Match State: " & state_array(state_name, item) & " -----")
-    		CALL write_bullet_and_variable_in_CASE_NOTE("Match State Active Programs", state_array(progs, item))
-    		CALL write_bullet_and_variable_in_CASE_NOTE("Match State Contact Info", state_array(contact_info, item))
-    	NEXT
-    	CALL write_variable_in_CASE_NOTE ("-----")
-    	CALL write_bullet_and_variable_in_CASE_NOTE("Client accessing benefits in other state", bene_other_state)
-    	CALL write_bullet_and_variable_in_CASE_NOTE("Contacted other state", contact_other_state)
-    	CALL write_bullet_and_variable_in_CASE_NOTE("Verification Requested", pending_verifs)
-    	CALL write_bullet_and_variable_in_CASE_NOTE("Verification Due", Due_date)
-    	CALL write_bullet_and_variable_in_CASE_NOTE("Other notes", other_notes)
-    	CALL write_variable_in_CASE_NOTE("----- ----- ----- ----- ----- ----- -----")
-    	CALL write_variable_in_CASE_NOTE ("DEBT ESTABLISHMENT UNIT 612-348-4290 EXT 1-1-1")
-
-ELSEIF clear_action_checkbox = CHECKED or notice_sent = "Y" THEN
-	IF sent_date <> "" then MsgBox("A difference notice was sent on " & sent_date & ". The script will now navigate to clear the PARIS match.")
+    		err_msg = ""
+    		Dialog Dialog1
+    		cancel_without_confirmation
+    		IF bene_other_state = "Select One:" THEN err_msg = err_msg & vbNewLine & "* Is the client accessing benefits in other state?"
+    		IF contact_other_state = "Select One:" THEN err_msg = err_msg & vbNewLine & "* Did you contact the other state?"
+    		IF fraud_referral = "Select One:" THEN err_msg = err_msg & vbnewline & "* You must select a fraud referral entry."
+    		IF err_msg <> "" THEN MsgBox "*** NOTICE!!! ***" & vbNewLine & err_msg & vbNewLine
+    	LOOP UNTIL err_msg = ""
+    	CALL check_for_password(are_we_passworded_out)
```

Comment:
Looks like this previous code had issues if there was a transmit - does that not pose a concern here anymore?

Reply:
Nope, this should stay on the same panel since it's not in edit mode and transmitting won't bring the user to another panel.